src/test/scala/com/monsanto/arch/cloudformation/model/FnGetAZs_UT.scala

```diff
@@ -0,0 +1,33 @@
+package com.monsanto.arch.cloudformation.model
+
+import org.scalatest.{FunSpec, Matchers}
+import spray.json._
+import DefaultJsonProtocol._
+
+class FnGetAZs_UT extends FunSpec with Matchers {
+
+  describe("Fn::GetAZs") {
+    it("should serialize correctly") {
```

Comment:
@kevinmeredith: In CFTG, we don't really care much about deserialization.  We often don't even implement it.